manifests/install.pp

```diff
@@ -65,6 +65,14 @@
 
   case $python::provider {
     pip: {
+      # Install pip without pip, see https://pip.pypa.io/en/stable/installing/.
+      exec { 'bootstrap pip':
+        command => 'curl https://bootstrap.pypa.io/get-pip.py | python',
```

Comment:
Downloading scripts from the internet and running them makes baby jesus cry. Can't we just install the packages?